specification/search/data-plane/Azure.Search/stable/2023-11-01/searchindex.json

```diff
@@ -0,0 +1,2084 @@
+            "name": "Full",
+            "description": "Uses the full Lucene query syntax for searches. Search text is interpreted using the Lucene query language which allows field-specific and weighted searches, as well as other advanced features."
+          },
+          {
+            "value": "semantic",
+            "name": "Semantic",
+            "description": "Best suited for queries expressed in natural language as opposed to keywords. Improves precision of search results by re-ranking the top search results using a ranking model trained on the Web corpus."
+          }
+        ]
+      },
+      "description": "Specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax."
+    },
+    "VectorQuery": {
+      "type": "object",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/VectorQueryKind",
+          "description": "The kind of vector query being performed.",
+          "x-nullable": false
+        },
+        "k": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of nearest neighbors to return as top hits."
+        },
+        "fields": {
+          "type": "string",
+          "description": "Vector Fields of type Collection(Edm.Single) to be included in the vector searched."
+        },
+        "exhaustive": {
+          "type": "boolean",
+          "description": "When true, triggers an exhaustive k-nearest neighbor search across all vectors within the vector index. Useful for scenarios where exact matches are critical, such as determining ground truth values."
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "description": "The query parameters for vector and hybrid search queries."
+    },
+    "RawVectorQuery": {
+      "type": "object",
+      "x-ms-discriminator-value": "vector",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorQuery"
+        }
+      ],
+      "properties": {
+        "vector": {
+          "type": "array",
+          "items": {
+            "type": "number",
+            "format": "float"
+          },
+          "description": "The vector representation of a search query."
+        }
+      },
+      "description": "The query parameters to use for vector search when a raw vector value is provided."
+    },
+    "VectorQueryKind": {
+      "type": "string",
+      "enum": [
+        "vector",
+        "text"
+      ],
+      "x-ms-enum": {
+        "name": "VectorQueryKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "vector",
+            "name": "Vector",
+            "description": "Vector query where a raw vector value is provided."
+          },
+          {
+            "value": "text",
+            "name": "Text",
+            "description": "Vector query where a text value that needs to be vectorized is provided."
+          }
```

Comment:
Shouldn't this be removed as support for text is added in `2023-10-01-Preview` which won't be part of this GA. This applies to `text` in the `enum` array as well.

Reply:
Good call, yes this is a mistake. Removing shortly. Thanks for catching this!

Reply:
This is removed now. Thanks again for catching this! :)